File: MultiLangPrograms/Python/Program1.py
"""Write a program to count the number of integers and characters 
in a given string. Input: "nAJh4837sj"""

def count_integers_and_characters(input_string):
    integer_count = 0
    character_count = 0

    for c in input_string:
        if c.isdigit():
            integer_count += 1
        elif c.isalpha():
            character_count += 1

    print(f"Number of integers: {integer_count}")
    print(f"Number of characters: {character_count}")

count_integers_and_characters("nAJh4837sj")

File: MultiLangPrograms/Java/Program1.java
/**
 * Write a program to count the number of integers and characters 
 * in a given string. Input: "nAJh4837sj"
 */

public class Program1 {
    public static void main(String[] args) {
        String input = "nAJh4837sj";
        int integerCount = 0;
        int characterCount = 0;

        for (int i = 0; i < input.length(); i++) {
            char c = input.charAt(i);
            if (Character.isDigit(c)) {
                integerCount++;
            } else if (Character.isLetter(c)) {
                characterCount++;
            }
        }

        System.out.println("Number of integers: " + integerCount);
        System.out.println("Number of characters: " + characterCount);
    }
}